torchtext/prototype/models/t5/modules.py

```diff
@@ -830,24 +832,44 @@ def __init__(
             ]
         )
         self.num_layers = num_layers
+        self.norm = T5LayerNorm(d_model)
+        self.dropout1 = nn.Dropout(dropout)
+        self.dropout2 = nn.Dropout(dropout)
```

Comment:
Why did we decide to move this from the model to the encoder/decoder?

Reply:
Keeps the entire encoder forward method self-contained.